### PR TITLE
Issue 149 - reading logs is done with bufio.NewReader instead of bufio.NewScanner

### PR DESCRIPTION
**Pull request by soltysh**
This addresses the problem described in #149.

### CODE REVIEW

**Comment by soltysh** [comment]
[test]

//cc @bparees @iocanel

**Comment by bparees** [comment]
lgtm.

File: pkg/build/strategies/layered/layered.go [hunk 1]
```diff
@@ -115,9 +115,16 @@ func (b *Layered) Build(request *api.Request) (*api.Result, error) {
 
 	// goroutine to stream container's output
 	go func(reader io.Reader) {
-		scanner := bufio.NewScanner(reader)
-		for scanner.Scan() {
-			glog.V(2).Info(scanner.Text())
+		scanner := bufio.NewReader(reader)
+		for {
+			if text, err := scanner.ReadString('\n'); err == nil {
```
Comment (mfojtik) [inline pkg/build/strategies/layered/layered.go]:
```
for {
  text, err := scanner.ReadString('\n')
  if err != nil {
    glog.Errorf("Error reading from Docker stdout: %v", err)
    break
  }
  glog.V(2).Info(text)
}
```

Comment (bparees) [inline pkg/build/strategies/layered/layered.go]: +1

**Comment by soltysh** [comment]
Reworked according to @mfojtik suggestion.

**Comment by bparees** [comment]
[merge]

**Comment by soltysh** [comment]
I've introduced a bug by doing `if err != nil && glog.V(2)` which resulted in integration testes exceeding maximal allowed time for the run - fixed.

**Pull request merged**